common/locales/en/front.json

```diff
@@ -97,24 +97,24 @@
+    "marketing2Lead1": "While you can play Habitica solo, the lights really turn on when you start collaborating, competing, and holding each other accountable. The most effective part of any self-improvement program is social accountability, and what better an environment for accountability and competition than a video game?",
     "marketing2Lead2": "<strong>Fight Bosses</strong>. What's a Role Playing Game without battles? Fight bosses with your party. Bosses are \"super accountability mode\" - a day you miss the gym is a day the boss hurts <em>everyone</em>.",
     "marketing2Lead2Title": "Bosses",
     "marketing2Lead3": "<strong>Challenges</strong> let you compete with friends and strangers. Whoever does the best at the end of a challenge wins special prizes.",
-    "marketing3Header": "Apps",
+    "marketing3Header": "Apps and Extensions",
     "marketing3Lead1": "The <strong>iPhone & Android</strong> apps let you take care of business on the go. We realize that logging into the website to click buttons can be a drag.",
     "marketing3Lead2": " Other <strong>3rd Party Tools</strong> tie Habitica into various aspects of your life. Our API provides easy integration for things like the <a href='https://chrome.google.com/webstore/detail/habitica/pidkmpibnnnhneohdgjclfdjpijggmjj?hl=en-US' target='_blank'>Chrome Extension</a>, for which you lose points when browsing unproductive websites, and gain points when on productive ones. <a href='http://habitica.wikia.com/wiki/App_and_Extension_Integrations' target='_blank'>See more here</a>",
     "marketing4Header": "Organizational Use",
-    "marketing4Lead1": "Education is one of the best sectors for gamification. We all know how glued to phones and games students are these days; harness that power! Pit your students against eachother in friendly competition. Reward good behavior with rare prizes. Watch their grades and behavior soar.",
+    "marketing4Lead1": "Education is one of the best sectors for gamification. We all know how glued to phones and games students are these days; harness that power! Pit your students against each other in friendly competition. Reward good behavior with rare prizes. Watch their grades and behavior soar.",
     "marketing4Lead1Title": "Gamification In Education",
-    "marketing4Lead2": "Health care costs are on the rise, and something's gotta give. Hundreds of programs are built to reduce costs and improve wellness. We believe Habitica can pave a substantial path towards healthy lifestyles.",
+    "marketing4Lead2": "Health care costs are on the rise, and something's gotta give. Hundreds of programs are built to reduce costs and improve wellness; we believe Habitica can pave a substantial path towards healthy lifestyles.",
```

Comment:
Hmmm, I like the period better. The period suggests that there's something that distinguishes Habitica from the hundreds of programs mentioned in the previous sentence. This separation is not clear with the semicolon. Additionally, I think short sentences are generally more effective in marketing language.